packages/ui-module/components/style.js

```diff
@@ -0,0 +1,25 @@
+import { createStyleSheet } from '@platzily-ui/styling';
+
+const useStyleSheet = createStyleSheet((theme, props) => {
+  return {
+    root: {
+      display: 'flex',
+      justifyContent: 'center'
+    },
+    box: {
+      backgroundColor: props.color || theme.palette.primary.main,
+      width: 200,
+      height: 200
+    }
+  };
+}, { key: 'StyledBox' });
+
+export default function StyledBox(props) {
```

Comment:
What are we expecting from this component? If the reasong is for us to have something in the index page while we develop the other features we can add this to the index page and remove this one here. If this component exist because of what I'm thinking it will be removed anyway right?

Reply:
I added this component to have something like reference to indicate where We have to add the components in the index page and to have something in the page while We develop other features but you're right I agree with your comment.

I will add an element inside 'index.js' and I will remove this file.